From 464c2f8b61855cb22d61c4b232f74d6767fac5fb Mon Sep 17 00:00:00 2001
From: Kai-Heng Feng <kai.heng.feng@canonical.com>
Date: Mon, 3 Aug 2020 23:57:45 +0800
Subject: [PATCH 11/32] conf: USB-Audio: Disable IEC958 on Lenovo ThinkStation
 P620

Both USB audio cards on Lenovo ThinkStation P620 don't support IEC958,
so disable IEC958 accordingly.

Signed-off-by: Kai-Heng Feng <kai.heng.feng@canonical.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 src/conf/cards/USB-Audio.conf | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/conf/cards/USB-Audio.conf b/src/conf/cards/USB-Audio.conf
index 297629f34d77..9b64af3c0da9 100644
--- a/src/conf/cards/USB-Audio.conf
+++ b/src/conf/cards/USB-Audio.conf
@@ -63,6 +63,8 @@ USB-Audio.pcm.iec958_device {
 	"Scarlett 2i4 USB" 999
 	"Sennheiser USB headset" 999
 	"SWTOR Gaming Headset by Razer" 999
+	"ThinkStation P620 Main" 999
+	"ThinkStation P620 Rear" 999
 	"Thunderbolt Dock Audio Headset" 999
 	"Thunderbolt Dock Audio Module" 999
 	"USB Device 0x46d_0x821" 999
-- 
2.16.4

